core/core.go

```diff
@@ -0,0 +1,2 @@
+// Package core contains the most-common System implementations according to the engo.io/ecs definition.
```

Comment:
That is exactly what I'm referring to. Thank you @mewmew - I'm surprised I even did it right (by mistake)